src/ui/public/chrome/directives/loading_indicator/loading_indicator.html

```diff
@@ -0,0 +1,7 @@
+<div
+  class="loadingIndicator"
+  ng-show="chrome.httpActive.length"
+  data-test-subj="globalLoadingIndicator"
```

Comment:
We use `data-test-subj` to identify specific parts of our UI that play a role from the perspective of our functional tests (and because functional tests mimic a user, you could say that these parts of the UI play the same roles for our users).  By using this attribute instead of a class, we can uncouple the way we've designed our functional tests from the way we've designed our components. One benefit to this is we can rename classes, or swap our components (which behave differently but perform the same function), without having to worry so much about breaking our functional tests.

I don't know what discussion led to the attribute name but I know it means "test subject", i.e. an object to be tested.